.github/ISSUE_TEMPLATE/incident.yml

```diff
@@ -0,0 +1,30 @@
+name: Incident 🚒
+description: Respond to a production incident - checklist
+labels: ["🚒 incident"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        A production incident started. Please stay cool, no need to call 911. Follow the checklist below - you've got this :muscle: <br/>
+        You may refer to the [Wing maintainers operational guide](https://www.winglang.io/docs/contributors/maintainers/operations) for more details.
+  - type: markdown
+    attributes:
+      value: |
+        ## Incident Response Checklist
+  - type: checkboxes
```

Comment:
Can you share a screenshot of how this will look:
1. when creating the issue?
2. after the issue is created.

I suspect the checklist form is not what we want but rather just some text that contains markdown checklists (e.g. `- [ ] foo`).

Reply:
I checked and it should actually remain as I initially did.
Screenshot when creating the issue:
<img width="659" alt="image" src="https://github.com/winglang/wing/assets/106860404/ab90a03b-5a92-4551-95b6-8f9e67f51a24">
Screenshot after the issue is created:
<img width="623" alt="image" src="https://github.com/winglang/wing/assets/106860404/125e88c8-0fae-41de-a55a-cca10ba04b2b">